### PR TITLE
Fix doctolib konnector

### PR DESCRIPTION
**Pull request by ZeHiro**

### CODE REVIEW

File: server/konnectors/doctolib.js [hunk 1]
```diff
@@ -176,23 +176,23 @@ function saveEvents(requiredFields, entries, data, next) {
...
         // Create the event.
         connector.logger.info('Creating event');
         Event.create(icalEvent, (err) => {
           if (err) {
-            next(err);
+            callback(err);
           }
           entries.nbCreations++;
-          next();
+          callback();
```
Comment (frankrousseau) [inline server/konnectors/doctolib.js]: Be careful, her the callback is called twice if an error occurs.

Comment (ZeHiro) [inline server/konnectors/doctolib.js]: @frankrousseau 
Shall I return calling callback(err)?

Comment (frankrousseau) [inline server/konnectors/doctolib.js]: Yep

**Comment by frankrousseau** [comment]
Thx @ZeHiro

**Pull request merged**